### ISSUE TITLE
VIRUS

### ISSUE THREAD

**Issue by Zawaer** [bacontsu/multibox#3]
Firefox says; "This file contains a virus or malware."

Just give the source code

**Comment by bacontsu** [comment]
> Firefox says; "This file contains a virus or malware."
> 
> Just give the source code

about the virus thingy I alr put it on the desc, also actually I alr mentions this thing a while ago here :
https://github.com/bacontsu/multibox/issues/1
https://www.virustotal.com/gui/url/c8faae3d8246125a7259f3d4b0f534511cf61ad4674458004b4869ac7f6dddf8/detection
at this point I'm pretty sure this is just false flagging by firefox, well not just firefox, sometimes (rarely) my win defender and chrome also acting up when I have this program opened

I still don't know why it's detecting as a virus, the abandoned source code I get it from doesn't flag anything about this. aswell as my other tools that based on this source code doesn't flag anything about virus  
https://github.com/bacontsu/autofarmer 
(but this one without the multibox fixed and also a bit 'heavier' because I've implemented more advanced autofarming on it, still 0% on full usage tho}



about source code, politely, no.
people will skid this program and put actual stealer inside, I don't actually need credits for this program, I just don't wanna people stealing me and my friends save.dat so I 'made' my on multibox, well I also kinda skidded this from Blacklock's source code except I'm trying to fix it, not putting anything suspicious on it. 

Not related, but you asking for source code when your account is made like 1 day before you make this comment is also kinda fishy ngl. 

i was thinking about uploading the source code on this page before but I decided to not do that, pretty sure many people will skid it or even doing something worse, like making an autoban trap.
